Ladybird/Android/src/main/java/org/serenityos/ladybird/LadybirdActivity.kt

```diff
@@ -9,17 +9,18 @@ package org.serenityos.ladybird
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import org.serenityos.ladybird.databinding.ActivityMainBinding
+import kotlin.io.path.Path
 
 class LadybirdActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityMainBinding
-    private lateinit var resourceDir: String
+    lateinit var resourceDir: String private set
```

Comment:
```suggestion
    lateinit var resourceDir: String
        private set
```

Reply:
That's what the Kotlin docs showed, so I was surprised that the auto-formatter didn't do that.